src/CalculateAdvantageFiscal.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: wilder
+ * Date: 22/11/18
+ * Time: 17:20
+ */
+namespace App;
```

Comment:
pourquoi vous vous lancez sur cet US dès maintenant ? ça devrait attendre. Là vous me mettez le fichier direct dans src ? pourquoi ? ça place sera dans un service, que vous verrez lundi. 
La US 5 porte sur la mise en place du formulaire, pas du calcul de l'avantage fiscal. Ca doit venir dans une autre US.

Reply:
L'avantage fiscal est rattaché à l'US 5 et US 6, la 5 car c'est selon les chiffres saisies dans la 5 que l'on a un résultat dans la 6.
Noté on doit donc lui crée une US à part entière même si rattachée à US 5 et 6.